docs/tech-reference/yum-plugins.rst

```diff
@@ -627,3 +627,13 @@ Optional Configuration Parameters
 ``force_full``
 Boolean flag to indicate whether or not publish should be done from scratch.
 If unspecified the incremental publish will be performed when possible.
+
+``remove_old_repodata``
+Boolean flag to indicate whether or not repodata passed expiration are removed
+during publish. Files that present in repomd.xml are preserved no matter of 
```

Comment:
Can you add two sentences in the middle here:  `If not present it defaults to False. The default expiration is 14 days.`